udata_front/theme/gouvfr/assets/js/components/MarkdownEditor/InternalEditor.vue

```diff
@@ -0,0 +1,222 @@
+<template>
+  <div class="editor__container">
+    <div class="fr-mx-auto fr-grid-row fr-pb-1v" role="menubar">
+        <div role="group" class="fr-pr-1w">
+          <EditorButton
+          icon="fr-icon-arrow-go-back-line"
+          @click="() => call(undoCommand.key)"
+          :title="t('Undo')"
+          />
+          <EditorButton
+            icon="fr-icon-arrow-go-forward-line"
+            @click="() => call(redoCommand.key)"
+            :title="t('Redo')"
+          />
+        </div>
+        <div role="separator" class="w-1px bg-grey-125 fr-my-1v">
+        </div>
+        <div role="group" class="fr-pr-1w">
+          <EditorButton
+            icon="fr-icon-bold"
+            @click="() => call(toggleStrongCommand.key)"
+            :title="t('Bold')"
+          />
+          <EditorButton
+            icon="fr-icon-italic"
+            @click="() => call(toggleEmphasisCommand.key)"
+            :title="t('Italic')"
+          />
+          <EditorButton
+            icon="fr-icon-h-3"
+            @click="() => call(wrapInHeadingCommand.key, 3)"
+            :title="t('Title size 3')"
+          />
+          <EditorButton
+            icon="fr-icon-h-4"
+            @click="() => call(wrapInHeadingCommand.key, 4)"
+            :title="t('Title size 4')"
+          />
```

Comment:
Really minor stuff: is there a reason to only expose title size 3 and 4? (maybe because h1 and h2 are taken for the page header?) But we may want to create a textarea with different values, no? And I think it may be a little bit confusing for users to understand why. We could hide this information (always presents the buttons starting from 1 but in the editor linking them to the correct values for the correct markup/style)

Reply:
Yes, H1 is the page title, H2 is the `description` label and then below we have the dataset / reuse / organization description.

I think we can add more logic based on props later if we have a case for a markdown editor outside of a `display` page (e.g. when we editorialize search pages) but this setting cover all 3 publishing forms.

We can show Title 1 and Title 2 instead of 3 and 4, this is indeed what's on Figma.